src/WebJobs.Script/Description/Workers/WorkerFunctionInvoker.cs

```diff
@@ -91,11 +91,23 @@ protected override async Task<object> InvokeCore(object[] parameters, FunctionIn
             };
 
             string invocationId = context.ExecutionContext.InvocationId.ToString();
-            _logger.LogTrace($"Sending invocation id:{invocationId}");
-            await _functionDispatcher.InvokeAsync(invocationContext);
-            var result = await invocationContext.ResultSource.Task;
+            ScriptInvocationResult result = new();
+
+            try
+            {
+                _logger.LogTrace("Sending invocation id: '{id}", invocationId);
+
+                await _functionDispatcher.InvokeAsync(invocationContext);
+                result = await invocationContext.ResultSource.Task;
+
+                await BindOutputsAsync(triggerValue, context.Binder, result);
+            }
+            catch (Exception ex) when (invocationContext.ResultSource.Task.IsCanceled)
```

Comment:
Do we want to check for `invocationContext.CancellationToken.IsCancellationRequested` instead (or as well)?

I think at this stage (in this code path), if the cancellation token is cancelled and there is an exception, it is during a function invocation path